operations/README.md

```diff
@@ -11,6 +11,13 @@ This section describes all operations supported by the API, organised here by th
 | [Add addresses](addresses.md#add-addresses) | Adds a new address to the system and assigns it to a specified account |
 | [Update addresses](addresses.md#update-addresses) | Updates an existing address in the system assigned to a specified account |
 
+## Cancellation policies
```

Comment:
This should be added under ## Services rather than creating a new section for ## Cancellation policies.